src/coreclr/jit/llvmdebuginfo.cpp

```diff
@@ -355,12 +354,12 @@ class TypeDebugInfoModule
 
     DIType* EmitEnumType(CORINFO_LLVM_ENUM_TYPE_DEBUG_INFO* pInfo)
     {
-        llvm::SmallVector<Metadata*, 24> elements(static_cast<size_t>(pInfo->ElementCount));
+        llvm::SmallVector<Metadata*, 32> elements(static_cast<size_t>(pInfo->ElementCount));
```

Comment:
The `SmallVector<T>` constructor, unlike e. g. `List<T>::.ctor`, accepts size and not capacity. So we were construction an array like `{null, null, null, elem0, elem1, elem2}` instead of `{elem0, elem1, elem2}`.